### PR TITLE
adds datastore sample using diplomat

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: datastore/lib/datastore_samples.ex [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      iex> GoogleApi.Datastore.Samples.create_task("Get Milk", "2% is best")
+
+  """
+  def create_task(name, description \\ "") do
+    Diplomat.Entity.new(
+      %{"description" => description},
+      "Task2",
+      name
+    ) |> Diplomat.Entity.insert
+    :ok
```
Comment (chingor13) [inline datastore/lib/datastore_samples.ex]: Seems weird to just throw away any response from the insert and just return `:ok`

Comment (bshaffer) [reply datastore/lib/datastore_samples.ex]: I agree. The response was `[nil]`, so I didn't know what to do with it.

Comment (bshaffer) [reply datastore/lib/datastore_samples.ex]: If you have a better way to handle it LMK, otherwise I'll merge!

Comment (dazuma) [reply datastore/lib/datastore_samples.ex]: That's definitely a bizarre response from Diplomat. My read of the source suggests that's what was sent back by the underlying RPC as the list of mutation responses. But I'm not comfortable relying on that, so I'll not advocate pattern matching on it. So yeah, uhh, LGTM or whatever.

**Pull request merged**